### PR TITLE
Adds PYTHONSAFEPATH to python_stub_template.txt

### PR DESCRIPTION
**Pull request by groodt**
See: https://github.com/bazelbuild/rules_python/issues/382

### CODE REVIEW

File: src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt [hunk 1]
```diff
@@ -314,6 +314,10 @@ def Main():
   if runfiles_envkey:
     new_env[runfiles_envkey] = runfiles_envvalue
 
+  # Don't prepend a potentially unsafe path to sys.path
+  # See: https://docs.python.org/3.11/using/cmdline.html#envvar-PYTHONSAFEPATH
+  new_env['PYTHONSAFEPATH'] = 'non_empty'
```
Comment (hrfuller) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt:319]: Using "1" or "True" would be clearer IMHO

Comment (groodt) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt:319]:
```suggestion
  new_env['PYTHONSAFEPATH'] = '1'
```

File: src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt [hunk 2]
```diff
@@ -314,6 +314,10 @@ def Main():
   if runfiles_envkey:
     new_env[runfiles_envkey] = runfiles_envvalue
 
+  # Don't prepend a potentially unsafe path to sys.path
```
Comment (rickeylev) [inline src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt:317]: fwiw, i'd recommend also linking to the rules_python issue. The unsafe behavior this is trying to prevent isn't immediately obvious.
(apologies for the drive-by comment)

Comment (groodt) [reply src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt:317]: 👍 Good idea.

**Review by comius — APPROVED** [review]
